flake.nix

```diff
@@ -1,50 +1,125 @@
 {
-  description = "Generate PureScript data types from Haskell data types";
```

Comment:
Tbh I think you'd be better of c/p-ing the Nix setup in Cardax. But also, you can leave Nixification for some other day, we can tackle it together, and just focus on the Haskell side of things.